sonar-plugin-api/src/main/java/org/sonar/api/utils/DateUtils.java

```diff
@@ -72,7 +73,7 @@ public static String formatDateTime(long ms) {
    * @since 6.6
    */
   public static String formatDateTime(OffsetDateTime dt) {
-    return DATETIME_FORMATTER.format(dt);
+    return DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ssxxx").format(dt);
```

Comment:
It should be better to have a constant, to understand more easily what is this format